plugins/woocommerce/tests/api-core-tests/global-setup.js

```diff
@@ -155,9 +154,15 @@ module.exports = async ( config ) => {
 				await setupPage
 					.locator( '#install-plugin-submit' )
 					.click( { timeout: 60000 } );
-				await setupPage.waitForLoadState( 'networkidle' );
+				await setupPage.waitForLoadState( 'networkidle', {
+					timeout: 60000,
+				} );
 				await expect(
-					setupPage.getByRole( 'link', { name: 'Activate Plugin' } )
+					setupPage.getByRole(
+						'link',
+						{ name: 'Activate Plugin' },
+						{ timeout: 60000 }
+					)
```

Comment:
Doesn't really matter running on CI/GH, but this was occasionally timing out for me locally (because I have a terrible upload speed at home).  So I bumped the timeouts.